operations/http.go

```diff
@@ -29,6 +29,7 @@ type legacyClient struct {
 	httpClient         http.Client
 	User               string
 	APIKey             string
+	JWT                string
```

Comment:
Since this JWT is specifically embedding it for the kanopy token header, could we rename it (or include a comment) to signify that?